compiler/src/main/kotlin/io/github/apl_cornell/viaduct/errors/InsecureDataFlowError.kt

```diff
@@ -21,23 +26,31 @@ class InsecureDataFlowError(
 
     override val description: Document
         get() {
-            if (!nodeLabel.confidentiality().flowsTo(to.confidentiality())) {
+            if (!context.lessThanOrEqualTo(
+                    nodeLabel.confidentialityComponent,
+                    to.confidentialityComponent
+                )
+            ) {
                 // Confidentiality is the problem
                 return Document("This term is flowing to a place that does not have enough confidentiality:")
                     .withSource(node.sourceLocation) /
                     Document("The term's confidentiality label is:")
-                        .withData(nodeLabel.confidentiality()) /
+                        .withData(nodeLabel.confidentiality(FreeDistributiveLattice.bounds())) /
                     Document("But it is going to a place that only guarantees:")
-                        .withData(to.confidentiality())
+                        .withData(to.confidentiality(FreeDistributiveLattice.bounds()))
             } else {
-                // Integrity is the problem
-                assert(!nodeLabel.integrity().flowsTo(to.integrity()))
+                assert(
```

Comment:
Same as above. Preserve old logic and use `Label` level functions.

Reply:
Added A label level actsfor function that takes congruence so that this can be implemented